AEPMessaging/Tests/FunctionalTests/MessagingNotificationTrackingTests.swift

```diff
@@ -71,18 +70,28 @@ class MessagingNotificationTrackingTests: FunctionalTestBase {
     
     func test_notificationTracking_whenUser_tapsNotificationBody() {
         // setup
+        var actualStatus : PushTrackingStatus?
+        let expectation = XCTestExpectation(description: "Messaging Push Tracking Response")
         setExpectationEvent(type: EventType.edge, source: EventSource.requestContent, expectedCount: 1)
         let response = prepareNotificationResponse()!
         
         // test
-        Messaging.handleNotificationResponse(response)
+        Messaging.handleNotificationResponse(response,closure: { status in
+            actualStatus = status
+            expectation.fulfill()
+        })
+        
+        // verify tracking status value
+        wait(for: [expectation], timeout: 2)
+        XCTAssertEqual(actualStatus, .trackingInitiated)
```

Comment:
nit: don't need to change here, but in general assert parameters are ordered as (expectedValue, actualValue)